docs/DEVGUIDE.md

```diff
@@ -174,137 +178,248 @@ The images are tagged with the current Git commit SHA:
-    kubectl create -f backend.yaml
+----
 
-    # Create binding called 'database'.
-    kubectl create -f binding.yaml
+Because we haven't created any resources in the service-catalog API server yet,
+`kubectl get` will return an lists of resources:
 
-Creating the binding will create a `Secret` for binding consumption. This
-can now be used by a native Kubernetes app. You can inspect the config map
-using `kubectl get secret database`:
-
-    NAME       DATA      AGE
-    database   4         55s
-
-Now you can deploy the application that consumes the binding:
-
-    kubectl create -f ../user-bookstore-client/bookstore.yaml
-
-This will create a Kubernetes service `user-bookstore-fe`. Wait for deployments
-to start and an IP address of the frontend to be assigned. You can monitor the
-external IP address creation using `kubectl get services`:
-
-    NAME                    CLUSTER-IP       EXTERNAL-IP       PORT(S)    AGE
-    cf-i-3a121d22-booksbe   10.107.254.221   <none>            3306/TCP   2m
-    user-bookstore-fe       10.107.254.221   **<pending>**         3306/TCP   2m
-    kubernetes              10.107.240.1     <none>            443/TCP    1d
-
-Once the IP address becomes available we can use it to contact the frontend
-endpoint. In this example, the IP address is `104.154.153.120`.
-
-Save the IP address in an environment variable:
-
-    IP=104.154.153.120
-
-And interact with the Bookstore API:
+    kubectl get brokers,serviceclasses,instances,bindings
 
-    # List shelves
-    curl "http://${IP}:8080/shelves"
+### Registering a Broker
 
-    # List a specific shelf
-    curl "http://${IP}:8080/shelves/1"
+First, we'll register a service broker with the catalog.  To do this, we'll
+create a new [`Broker`](../contrib/examples/walkthrough/ups-broker.yaml)
+resource:
 
-    # Create a new shelf
-    curl -H 'Content-Type: application/json' \
-         -H 'x-api-key: 123' \
-          -d '{ "theme": "Travel" }' \
-          "http://${IP}:8080/shelves"
+    kubectl create -f contrib/examples/walkthrough/ups-broker.yaml
+    broker "ups-broker" created
 
-    # Create a book on the shelf:
-    curl -H 'Content-Type: application/json' \
-         -H 'x-api-key: 123' \
-         -d '{ "author": "Rick Steves", "title": "Travel as a Political Act" }' \
-         "http://${IP}:8080/shelves/3/books"
+Kubernetes APIs are intention based; creating this resource indicates that the
+want for the service broker it represents to be consumed in the catalog.  When
+we create the resource, the controller handles loading that broker into the
+catalog by seeing what services it provides and adding them to the catalog.
 
-    # List the books on the travel shelf:
-    curl -H 'x-api-key: 123' "http://${IP}:8080/shelves/3/books"
+We can check the status of the broker using `kubectl get`:
 
-    # Get the book:
-    curl -H 'x-api-key: 123' "http://${IP}:8080/shelves/3/books/3"
+    kubectl get brokers ups-broker -o yaml
```

Comment:
It might be good to show the output of this command and point out the status section since you're talking about "status of the broker".  Basically directing people to the key info of interest.